src/dal/Jalasoft.TeamUp.Resumes.DAL/ResumeSQLRepository.cs

```diff
@@ -45,10 +45,26 @@ public void Delete(Guid id)
 
         public IEnumerable<Resume> GetAll()
         {
-            IEnumerable<Resume> resumes = new List<Resume>();
+            IEnumerable<Resume> resumes;
             using (IDbConnection db = new SqlConnection(this.connectionString))
             {
-                resumes = db.Query<Resume>("SELECT Id, Title, Sumary, CreationDate, LastUpdate FROM Resume");
+                string sql = "select * from Resume r join Person p on r.IdPerson = p.Id join Contact c on r.IdContact = c.Id join Resume_Skill rs on r.Id = rs.IdResume join Skill s on rs.IdSkill = s.Id ";
```

Comment:
please user upper case for all sql commands and specify the field needed instead of using *